qiskit/quantum_info/states/stabilizerstate.py

```diff
@@ -259,7 +259,34 @@ def evolve(
         ret._data = self.clifford.compose(other.clifford, qargs=qargs)
         return ret
 
-    def expectation_value(self, oper: Pauli, qargs: None | list = None) -> complex:
+    def expectation_value(self, oper: Pauli | SparsePauliOp, qargs: None | list = None) -> complex:
+        """Compute the expectation value of a Pauli or SparsePauliOp operator.
+
+        Args:
+            oper (Pauli or SparsePauliOp): a Pauli or SparsePauliOp operator to evaluate expval.
+            qargs (None or list): subsystems to apply the operator on.
+
+        Returns:
+            complex: the expectation value.
```

Comment:
The docs-typehints can be omitted since we already have them in the signature (then there's no danger of one of them getting out-of-date)
```suggestion
            oper: A Pauli or SparsePauliOp operator to evaluate the expectation value.
            qargs: Subsystems to apply the operator on.

        Returns:
            The expectation value.
```